doc/classes/Geometry3D.xml

```diff
@@ -73,6 +73,17 @@
 				Given the two 3D segments ([param p1], [param p2]) and ([param q1], [param q2]), finds those two points on the two segments that are closest to each other. Returns a [PackedVector3Array] that contains this point on ([param p1], [param p2]) as well the accompanying point on ([param q1], [param q2]).
 			</description>
 		</method>
+		<method name="get_triangle_barycentric_coords">
+			<return type="Vector3" />
+			<param index="0" name="point" type="Vector3" />
+			<param index="1" name="a" type="Vector3" />
+			<param index="2" name="b" type="Vector3" />
+			<param index="3" name="c" type="Vector3" />
+			<description>
+				Returns a [Vector3] containing weights based on how close a 3D position ([param point]) is to a triangle's different vertices ([param a], [param b] and [param c]). This is useful for interpolating between the data of different vertices in a triangle. One example use case is using this to smoothly rotate over a mesh instead of relying solely on face normals.
+				[url=https://en.wikipedia.org/wiki/Barycentric_coordinate_system]Here is a more detailed explanation of barycentric coordinates.[/url]
+			</description>
+		</method>
```

Comment:
This should likely mention:
- what's a valid input (`a`, `b`, `c` forming a triangle and `point` being coplanar with the `abc` triangle),
- what's the behavior for an invalid input (`a`, `b`, `c` not forming a valid triangle, or `point` being not coplanar with the `abc` triangle).

Reply:
Let's delegate this to a follow-up. This has been cooking long enough :)

Reply:
I can open a new PR for this if an issue is created.

Reply:
@PrecisionRender No, just link back to this comment discussion and the PR